### ISSUE TITLE
[Improvement]: Improve the display of `Duration` in the page of `Optimizing>Tables`

### ISSUE THREAD

**Issue by zhongqishang** [apache/amoro#2519]
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

Improve the display of `Duration` in the page of `Optimizing>Tables`, same as with Optimizing process detail page. 

![image](https://github.com/NetEase/amoro/assets/16421989/2a0b0a68-9173-41e4-88ef-eeb0fbdf33a4)

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by old-fish-567** [comment]
Kindly to ask if there is somebody working for this issue? If not, I'm glad to undertake it.

**Comment by old-fish-567** [comment]
@zhongqishang

**Comment by zhongqishang** [comment]
> Kindly to ask if there is somebody working for this issue? If not, I'm glad to undertake it.

@old-fish-567 Welcome and congratulations, assigned to you.

**Comment by Jzjsnow** [comment]
Hi @zhongqishang , I noticed that the `Duration` on the `Optimizing > Tables` page is displayed using `formatMS2DisplayTime`, while the `Duration` in Optimizing process detail page uses `formatMS2Time`. 

https://github.com/apache/amoro/blob/aa379e276055ea4faa9b18d0da60b4525af9fe64/amoro-web/src/utils/index.ts#L57-L126


Meanwhile, when hovering over the `Duration` on the `Optimizing > Tables` page, the displayed result in the hover box is the “formatMS2Time” formatted result, which is the same format as the duration display on the Optimization Process Detail page.

<img width="1228" height="596" alt="Image" src="https://github.com/user-attachments/assets/88da7ba0-ad02-4eec-bcef-0becc0a999fd" />

Could you please clarify the purpose and intent behind this optimization? I am a bit confused because the initial design might have intended to display these durations differently, which is why different methods were used.

**Comment by zhongqishang** [comment]
> Could you please clarify the purpose and intent behind this optimization? I am a bit confused because the initial design might have intended to display these durations differently, which is why different methods were used.

@Jzjsnow It is better to display a readable and friendly format directly without hovering. Another point is to keep it consistent with the time formatting of other pages (Optimizing Page). WDYT?

<img width="464" height="117" alt="Image" src="https://github.com/user-attachments/assets/55da1c7a-3ed1-449e-a270-c9afce3f2196" />

**Comment by Jzjsnow** [comment]
I agree that it's more friendly to display the time in a readable way on the page. I think we can swap what is displayed in the table and the hover box, so that if the user wants to see what was displayed before, they can hover over it with the mouse.

Before:
<img width="2286" height="669" alt="Image" src="https://github.com/user-attachments/assets/27b13106-f647-48ad-ab53-0f212aaed336" />

After:
<img width="2320" height="585" alt="Image" src="https://github.com/user-attachments/assets/ff3fea0b-0e27-40a0-83b3-6a74be434e14" />
